src/Models/Model.php

```diff
@@ -261,4 +278,34 @@ public function polymorphicManyToManyTables(): array
     {
         return $this->polymorphicManyToManyTables;
     }
+
```

Comment:
When you open a PR to add this for `Controller`, please extract all this into a trait. Maybe something like `ClassDefinition`.

Reply:
should I do it in this PR or should I wait for this merge first and then do another PR in which I put everything under a trait?

Reply:
Yes, after this is merged you can refactor in the next PR.